config/rbac/role.yaml

```diff
@@ -139,6 +139,14 @@ rules:
   - placements/finalizers
   verbs:
   - update
+- apiGroups:
```

Comment:
These role changes need to be added to `config/dr-cluster/rbac/role.yaml` as well. As the dr-cluster is bundled with the role here.

Reply:
Done